.prettierrc

```diff
@@ -1 +1 @@
-
+{}
```

Comment:
hmm was this giving your issues? .prettierrc can be either yaml OR json so I'm curious why the blank file didn't work for you 🤔

Reply:
yeah this was actually, I'm not sure why.  I am using vscode and it wouldn't format unless i had something here.  maybe i can take a look at why or remove this once the PR is actually done

Reply:
[This](https://github.com/prettier/prettier-vscode/issues/570#issuecomment-420694808) is why.

It also doesn't work with `.yml` or `.yaml` if completely empty.